com.unity.formats.fbx/Editor/FbxExportSettings.cs

```diff
@@ -1170,8 +1224,15 @@ internal static bool CanInstall()
 
         internal static string GetAbsoluteSavePath(string relativePath){
             var absolutePath = Path.Combine(Application.dataPath, relativePath);
-            return NormalizePath(absolutePath, isRelative: false,
-                separator: Path.DirectorySeparatorChar);
+            absolutePath = NormalizePath(absolutePath, isRelative: false, separator: Path.DirectorySeparatorChar);
+            
+            // check if path is outside Assets folder
+            if (string.IsNullOrEmpty(ExportSettings.ConvertToAssetRelativePath(absolutePath)))
+            {
+                return relativePath;
```

Comment:
shouldn't it return absolutePath here?

Reply:
No, the naming is just a bit confusing. I'll rename the variables, so it makes more sense.